server_test.go

```diff
@@ -674,10 +675,12 @@ func TestGlobalServerS3PluginFlush(t *testing.T) {
 
 	s3p := &s3p.S3Plugin{Logger: log, Svc: client}
 
+	pluginCount := len(server.getPlugins())
```

Comment:
In this case, I thought that the test a few lines down was intended to see whether registering the s3 plugin would increase the `pluginCount` by 1. I thought that having `pluginCount := len(server.getPlugins())` was more clear than something like
```
# pluginCount will be 1 by default because the flush_file defaults to /dev/null
pluginCount := 1
```